lints/lint_dnsname_contains_bare_iana_suffix.go

```diff
@@ -0,0 +1,43 @@
+// lint_dnsname_contains_bare_iana_suffix.go
```

Comment:
Cut this out.

And put in the license header.

Reply:
👍